e2e/_suites/fleet/features/fleet_mode_agent.feature

```diff
@@ -79,11 +79,11 @@ Examples:
 @reenroll
 Scenario Outline: Re-enrolling the <os> agent
   Given a "<os>" agent is deployed to Fleet with "tar" installer
-    And the agent is un-enrolled
-    And the "elastic-agent" process is "stopped" on the host
-  When the agent is re-enrolled on the host
+  When the agent is un-enrolled
+  Then the "elastic-agent" process is "stopped" on the host
```

Comment:
I believe this is part of the `When` clause, as it's what triggers the `Then` parts
```suggestion
    And the "elastic-agent" process is "stopped" on the host
```

Reply:
With current sequence, we are telling that if we un-enroll the agent, then the process is stopped. If that is correct, then your scenario is perfect.

Reply:
no, you're right - I believe I had only changed it as a guess to meet linting requirments, it is more accurate the way it was, I have revereted it.  And it passes linting, so it must have been changed during that period where I was unsure of the linting rules and I went overboard hoping to get it to pass.  it passes locally as such, if it fails in ci - then we have a mis-match in how the rules are being applied somehow, which we can follow up as needed.

to finish off that part of the convo, I broke linting to make sure I got some feedback... and I saw the local lint error, then I changed it to the syntax i've submitted with my last / latest comment and it is back to passing.

<img width="759" alt="Screen Shot 2020-11-30 at 4 00 20 PM" src="https://user-images.githubusercontent.com/12970373/100664666-f4df1800-3325-11eb-8d86-ef06f634667c.png">